src/analyze/analyze.c

```diff
@@ -2366,6 +2382,18 @@ static int parse_argv(int argc, char *argv[]) {
 
                         break;
 
+                case ARG_ALIAS: ;
+                        if (argc > 4)
```

Comment:
I don't think you want to check this here - one might be combining multiple --flags which should be fine. You should check at the end, with the number of arguments left after all flags have been parsed.